code/WorkInProgress/JobXPRewards.dm

```diff
@@ -547,7 +547,7 @@ mob/verb/checkrewards()
 /datum/jobXpReward/bartender/spectromonocle
 	name = "Spectroscopic Monocle"
 	desc = "Now you can look dapper and know which drinks you poisoned at the same time"
-	required_levels = list("Bartender"=5)
+	required_levels = list("Bartender"=0)
```

Comment:
Not part of the changelist. Suspect changed as part of debugging?

Reply:
I listed it in About this PR. I'll add it to the changelog